app/controllers/bets_controller.rb

```diff
@@ -0,0 +1,24 @@
+class BetsController < ApplicationController
+  def new
+    @bet = Bet.new
+  end
+
+  def create
+    @bet = Bet.new(bet_params)
+    if @bet.save
+      flash[:success] = 'Bet has been created'
+      redirect_to root_path
+    else
+      @bet.errors.full_messages.each do |message|
```

Comment:
Another way to do this is `@bet.errors.full_messages.to_sentence` will join all of your errors into a human readable string